integration_tests/2pc_test.go

```diff
@@ -218,6 +218,29 @@ func (s *testCommitterSuite) TestCommitRollback() {
 	})
 }
 
+func (s *testCommitterSuite) TestCommitOnTiKVDiskFullOpt() {
+	s.Nil(failpoint.Enable("tikvclient/rpcAllowedOnAlmostFull", `return("true")`))
```

Comment:
Should also add a situation that shows without `DiskFullOpt`, transaction will fail if the failpoint is enabled.

Reply:
Done